src/components/timeline/TrackThread.js

```diff
@@ -98,19 +96,13 @@ class TimelineTrackThread extends PureComponent<Props> {
     start: Milliseconds,
     end: Milliseconds
   ) => {
-    const {
-      rangeStart,
-      rangeEnd,
-      updatePreviewSelection,
-      changeSelectedThread,
-    } = this.props;
+    const { rangeStart, rangeEnd, updatePreviewSelection } = this.props;
     updatePreviewSelection({
       hasSelection: true,
       isModifying: false,
       selectionStart: Math.max(rangeStart, start),
       selectionEnd: Math.min(rangeEnd, end),
     });
-    changeSelectedThread(threadIndex);
```

Comment:
this is taken care of in https://github.com/devtools-html/perf.html/blob/2385360d481acf88f17b9503555289087a652b75/src/components/timeline/GlobalTrack.js#L166-L171